src/commands/blocks.ts

```
@@ -161,6 +162,7 @@ export const blocksCommand = define({
 			offline: ctx.values.offline,
 			sessionDurationHours: ctx.values.sessionLength,
 			timezone: ctx.values.timezone,
+			locale: ctx.values.locale,
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)